lib/Value.js

```diff
@@ -94,7 +104,9 @@ var Value = function (_React$Component) {
 				'span',
 				{ className: 'Select-value-icon',
 					'aria-hidden': 'true',
+					'tabIndex': 0,
```

Comment:
random question: why is aria-hidden=true on an element who can get focus with tabindex=0? This can result in a sort of nameless thingie when you focus on it.